PowerPointLabs/PowerPointLabs/CropLab/CropLabErrorHandler.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+
+namespace PowerPointLabs.CropLab
+{
+    internal class CropLabErrorHandler
+    {
+        private CropLabUIControl View { get; set; }
+        private static CropLabErrorHandler _errorHandler;
+        
+        public const string SelectionTypeShape = "shape";
+        public const string SelectionTypePicture = "picture";
+
+        public const int ErrorCodeSelectionIsInvalid = 0;
+        public const int ErrorCodeSelectionMustBeShape = 1;
+        public const int ErrorCodeSelectionMustBePicture = 2;
+        public const int ErrorCodeAspectRatioIsInvalid = 3;
+        public const int ErrorCodeSelectionCountZero = 4;
+        public const int ErrorCodeUndefined = 5;
+
+        private const string ErrorMessageSelectionIsInvalid = TextCollection.CropLabText.ErrorSelectionIsInvalid;
+        private const string ErrorMessageSelectionMustBeShape = TextCollection.CropLabText.ErrorSelectionMustBeShape;
+        private const string ErrorMessageSelectionMustBePicture = TextCollection.CropLabText.ErrorSelectionMustBePicture;
+        private const string ErrorMessageAspectRatioIsInvalid = TextCollection.CropLabText.ErrorAspectRatioIsInvalid;
+        private const string ErrorMessageUndefined = TextCollection.CropLabText.ErrorUndefined;
+        private const string ErrorMessageForSelectionCountZero = TextCollection.CropToSlideText.ErrorMessageForSelectionCountZero;
+
+        private CropLabErrorHandler(CropLabUIControl view = null)
```

Comment:
change the constructor to accept CropLabUIControl's interface, so that you can inject other its implementation, e.g. dummy ui control